job/validators.py

```diff
@@ -390,7 +397,7 @@ def cross_clean_pv(pvmodel):
 
             if self.models["Generator"].__getattribute__("replacement_year") == None:
                 if self.models["Settings"].off_grid_flag==False:
-                    self.models["Generator"].replacement_year = 25
+                    self.models["Generator"].replacement_year = 25 # TODO: set to analysis_years
```

Comment:
@rathod-b  this should be set to analysis_years when off_grid_flag is False, instead of 25.

Reply:
updated